.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+before:
+  hooks:
+    - go mod download
+builds:
+-
+  id: capdctl
+  env:
+    - CGO_ENABLED=0
+  main: ./cmd/capdctl/main.go
+  binary: capdctl
+  ldflags:
+    - "-s -w -X sigs.k8s.io/cluster-api-provider-docker/cmd/versioninfo.Version={{.Version}}"
+  goos:
+    - linux
+    - darwin
+  goarch:
+    - amd64
+-
+  id: capd-manager
+  env:
+    - CGO_ENABLED=0
+  main: ./cmd/capd-manager/main.go
+  binary: capd-manager
+  goos:
+    - linux
+  goarch:
+    - amd64
+archives:
+- replacements:
+    darwin: Darwin
+    linux: Linux
+    amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+dockers:
+  - goos: linux
+    goarch: amd64
+    binaries:
+    - capd-manager
+    image_templates:
+    - "gcr.io/kubernetes1-226021/capd-manager:v{{.Version}}"
```

Comment:
assuming `gcr.io/kubernetes1-226021` is not your gcr repo but a community-owned repo

Reply:
Don't know if this will be useful, but we might also want to use template variables for the image repository.

Reply:
yeah, this needs to get replaced with a community owned project. That's the perpetual issue with every kubernetes-sigs project. Will make an issue for that